infrastructures/openshift/src/main/java/org/eclipse/che/workspace/infrastructure/openshift/OpenShiftInternalRuntime.java

```diff
@@ -90,7 +90,6 @@ public OpenShiftInternalRuntime(
   @Override
   protected void internalStart(Map<String, String> startOptions) throws InfrastructureException {
     try {
-      project.cleanUp();
```

Comment:
Note that all objects in the project will be stopped even when the dedicated project is configured. See https://github.com/eclipse/che/pull/6400/files#diff-fd5e0f37b880417aed392d5dc7020dd6R149

Reply:
Yeah, I know. That's why I defined it as a "temp" solution. Basically, having this property for now will allow at least somehow work with spi on prod cluster. And in any case I'm not sure why `project.cleannup()` is called in internal startup (I guess it is 'just in case')